gnosis/safe-react
# Migrate Safe Apps endpoint to load directly from CGW

## What it solves
Resolves #3129

Now Safe Apps are loaded directly from CGW.

Removed config-service references.


## How this PR fixes it

Now Safe Apps are loaded directly from CGW.

## How to test it

1. Go to Safe Apps Section
2. See the network tab in the dev tools
3. The new CGW endpoint should be used to fetch Safe Apps List `https://safe-client.staging.gnosisdev.com/v1/chains/{chainId}/safe-apps`


## Screenshots
Old config-service endpoint
![Captura de pantalla 2021-12-20 a las 12 10 36](https://user-images.githubusercontent.com/26763673/146759418-8bb5498b-3cda-4f85-a181-c0c64963d61a.png)

New CGW endpoint
![Captura de pantalla 2021-12-20 a las 12 10 43](https://user-images.githubusercontent.com/26763673/146759426-d843c49e-7ac0-40c9-ad32-4c8aa9c7d3ea.png)
